src/commands/install/install.py

```diff
@@ -22,10 +25,19 @@ def install(
         raise installation_exceptions.InvalidLocalRepository()
 
     package_name = map_url_or_ssh_to_package_name(url_or_ssh)
+    package_dir = path.join(destination, package_name)
+
+    # pylint: disable=logging-fstring-interpolation
+    logger.info(
```

Comment:
Injecting color information by not using `fstring` seemed off to me. Color info probably can be injected like any other parameter.  I'll change that to avoid mixing interpolation types.